.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,7 @@ Resolves #XXXX
 - Admin permissions:
 
 ### How is this tested? (please write tests!) 💖💪
+Note: if you see a flake in your test build in github actions, please add to https://github.com/rubyforgood/casa/wiki/Flaky-tests or post in slack #casa
```

Comment:
> please add to https://github.com/rubyforgood/casa/wiki/Flaky-tests

How do you modify that page?

Reply:
hmm it may be a separate permission if you don't have an edit button :/